module_utils/network_lsr/argument_validator.py

```diff
@@ -612,7 +611,8 @@ def _validate_post(self, value, name, result):
 
 class ArgValidator_DictConnection(ArgValidatorDict):
 
-    VALID_STATES = ["up", "down", "present", "absent", "wait"]
+    VALID_STATES = ["up", "down"]
+    VALID_PERSISTENT_STATES = ["absent", "present"]
```

Comment:
Is it correct that "wait" has disappeared?

Reply:
Yes, it was not documented before and it just resulted in `time.sleep()`.